packages/editor/CodeMirror/utils/formatting/toggleSelectedLinesStartWith.ts

```diff
@@ -109,11 +117,28 @@ const toggleSelectedLinesStartWith = (
 			newSel = EditorSelection.range(fromLine.from, toLine.to + charsAdded);
 		}
 
+		let announcement = '';
+		if (charsAdded > 0) {
+			announcement = _('Added %s markup', accessibleName);
+		} else if (charsAdded < 0) {
+			announcement = _('Removed %s markup', accessibleName);
+		}
+
+		if (changedLineCount > 1) {
+			announcement += ` ${_('on %d lines', changedLineCount)}`;
+		} else {
+			announcement += ` ${_('to line start')}`;
+		}
```

Comment:
We cannot concatenate localised strings since the concatenation might not work in certain languages due to words being in a different order. 

I'm not too sure what's the solution here, but maybe simplifying the message would help even if we lose a bit of information. It would also make the work of translators easier.